backend/src/chats/chat/dto/create-chat.dto.ts

```diff
@@ -12,4 +13,7 @@ export class CreateChatDto {
 
 	@IsOptional()
 	type?: string;
+
+	@IsOptional()
```

Comment:
should this always have users? as in not optional?

Reply:
that would probably be good :P 
but since I kind of half-assed it while adding this functionality that would be a different ticket (made a ticket #162)